application.py

```diff
@@ -1 +0,0 @@
-from app import application
```

Comment:
Love that we can get rid of this.

Reply:
right? It bothered me right from the get go lol